rotkehlchen/data_migrations/migrations/migration_11.py

```diff
@@ -42,10 +47,45 @@ def _get_optimism_transaction_fees(rotki: 'Rotkehlchen', progress_handler: 'Migr
         )
 
 
+def _detect_arbitrum_one_accounts(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:  # noqa: E501
+    """
+    Detect accounts that have activity in other evm chains and are not yet tracked.
+    The reason is to start the new release with arbitrum one already detected.
+
+    Initially we check updates for spam assets and rpc nodes. Spam assets are updated before
+    accounts detection to avoid detecting accounts that only have spam assets. We also check
+    for rpc nodes updates here, to avoid an edge case where a call to rpc nodes happens before the
+    rpc node update task has finished.
+    """
+    progress_handler.new_step('Fetching new spam assets and rpc data info')
+    rotki.data_updater.check_for_updates(updates=[UpdateType.SPAM_ASSETS, UpdateType.RPC_NODES])
+
+    with rotki.data.db.conn.write_ctx() as write_cursor:
+        rotki.data.db.add_external_service_credentials(
+            write_cursor=write_cursor,  # add temporary etherscan arbitrum one key
+            credentials=[ExternalServiceApiCredentials(
+                service=ExternalService.ARBITRUM_ONE_ETHERSCAN,
+                api_key=ApiKey('VQUFYKKJR4RK8HFHYIJ9I93FIUVP44TN99'),  # same one in tests
+            )])
+
+    rotki.chains_aggregator.detect_evm_accounts(progress_handler)
```

Comment:
No need to run through all other chains. Especially if user has no activity there, and has never put etherscan api key this could take a long time.

```suggestion
    rotki.chains_aggregator.detect_evm_accounts(progress_handler, chains=SupportedBlockchain.ARBITRUM_ONE)
```